plugins/modules/host.py

```diff
@@ -217,7 +237,7 @@ def main():
             managed=dict(type='bool'),
             build=dict(type='bool'),
             ip=dict(),
-            mac=dict(),
+            mac=dict(type=mac),
```

Comment:
So this is how you add your own validation to ansible parameters?
You really always keep learning. ;)

Reply:
Yes, until it fucks up `validate-modules` :(